FEATURE_GATES.md

```diff
@@ -62,6 +62,7 @@ Features that reach GA and over time become stable will be removed from this tab
 | GatewayAlpha     | `false` | Alpha | 2.6.0   | TBD   |
 | ExpressionRoutes | `false` | Alpha | 2.10.0  | TBD   |
 | CombinedServices | `false` | Alpha | 2.10.0  | TBD   |
+| FillIDs          | `true`  | Beta  | 2.10.0  | 3.0.0 |
```

Comment:
Have we decided to remove `FillIDs` feature gate (and enable filling IDs) in 3.0.0?

Reply:
That's what we agreed with @mflendrich to go with. In the first place, we meant to introduce this change without a feature gate, but later we came to the conclusion that in some cases it may be better to allow opting out of it for some time. 

An example of a situation in which someone might find it not desired is having a huge setup, with many Services, Routes, and Consumers. When upgrading to KIC that fills deterministic IDs, it will mean all the objects that were generated previously without IDs, will have to be recreated with the new stable IDs. It's possible that in the case of DB-backed Gateways that may result in a temporary downtime when the first sync will take place after the upgrade as the deck sync is not atomic - the old objects will have to be removed first and new ones will be created afterward (at least that's what's going to happen if https://github.com/Kong/deck/pull/918 passes the review). 

@shaneutt has made a good point that we should document when users could potentially turn the feature off, I'll add more details to the changelog.

I'm happy to discuss more if you feel that's not the best approach.